query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/discover/LdapUserAttributes.java

```diff
@@ -104,6 +97,19 @@ public LdapUserAttributes() {
     return entries;
   }
 
+  private Set<String> getServerGuesserClaimAttribute(ServerGuesser serverGuesser) {
```

Comment:
It looks like Sonar was complaining about a try catch within another try catch. Instead of breaking this out into a separate method, can we put the try catch in the `ServerGuesser`'s `getClaimAttributeOptions` method?